ioSPI/cryo.py

```diff
@@ -338,3 +376,21 @@ def write_inp_file(dict_params, inp_file="input.txt"):
             "mtf_beta = {0[mtf_beta]}\n"
             "image_file_out = {0[image_file_out]}\n".format(dict_params["detector"])
         )
+
+
+def write_defocus_file(defocus_sample, defocus_file):
```

Comment:
This should respect the convention for ioSPI (discussed in the PR #46 just merged)

- first argument is where we will write, and it is called "path"
- second argument is what we will write